src/Routes.js

```diff
@@ -15,10 +15,15 @@ class Routes extends React.Component {
         <Switch>
           <Route exact path="/login" component={Login} />
           <Route exact path="/signup" component={Signup} />
-          <Route exact path="/main" component={Main} />
-          <Route exact path="/newproducts" component={NewProducts} />
-          <Route exact path="/hotproducts" component={HotProducts} />
-          <Route exact path="/mypage" component={MyPage} />
+          <Route exact path="/products/type=new" component={NewProducts} />
+          <Route exact path="/products/type=hot" component={HotProducts} />
```

Comment:
```suggestion
          <Route exact path="/products?type=new" component={NewProducts} />
          <Route exact path="/products/?type=hot" component={HotProducts} />
```
쿼리스트링으로 표현해야 할 부분인지 경로로 표현해야 할 부분인지 확인해주세요

Reply:
수정했습니다.